openlibrary/core/fulltext.py

```diff
@@ -17,16 +16,17 @@ def fulltext_search_api(params):
     search_select = search_endpoint + '?' + urllib.parse.urlencode(params, 'utf-8')
 
     try:
-        json_data = urllib.request.urlopen(search_select, timeout=30).read()
+        response = requests.get(search_select, timeout=30)
         logger = logging.getLogger("openlibrary.inside")
```

Comment:
Please move this line up into global scope so that we only `logging.getLogger()` once when this file is initialized.

Reply:
Also, please move `logger.debug('URL: ' + search_select)` _before_ the call to `requests.get()`.